langfuse/client.py

```diff
@@ -401,6 +416,14 @@ def task():
         except Exception as e:
             self.log.exception(e)
 
+    def end(self):
+        try:
+            end_time = datetime.now()
+            self.update(UpdateGeneration(endTime=end_time))
```

Comment:
I missed this one here: i think we need to return this function here so that end() returns a stateful client (also above). Otherwise good for me.